plugins/headlamp-plugin/test-headlamp-plugin.sh

```diff
@@ -53,3 +53,10 @@ npm run lint-fix
 
 # test type script error checks
 npm run tsc
+
+# test upgrade adds missing files
+rm jsconfig.json tsconfig.json src/headlamp-plugin.d.ts
+node bin/headlamp-plugin.js upgrade --no-package-updates
```

Comment:
The option seems to be called `skip-package-updates`.

Reply:
Thanks.